README.md

```diff
@@ -78,7 +78,7 @@ def print_array(array)
 end
 ```
 
-Snippet 7 - Big O:
+Snippet 7 - Big O: O(n) linear
```

Comment:
This one is actually O(n^2). The outer .each loop is O(n) and in the worst case the inner while loop is also O(n) --> multiplied together because nested is O(n^2)